src/main/CommandLine.cpp

```diff
@@ -878,6 +878,7 @@ runCatchup(CommandLineArgs const& args)
             config.RUN_STANDALONE = true;
             config.MANUAL_CLOSE = true;
             config.DISABLE_BUCKET_GC = disableBucketGC;
+            config.CATCHUP_SKIP_KNOWN_RESULTS = true;
```

Comment:
I think for now we should have operators manually opt-in

Reply:
Sorry that was just for testing purposes 😅